README.md

```diff
@@ -223,6 +223,15 @@ From there, you can use `GET` and `HEAD` requests with allt he same headers and
 
 Note that you cannot `PUT` or `DELETE` data in a versioned folder.
 
+### `fetch('hyper://NAME/$/diff/OLD_VERSION..NEW_VERSION/folder/')`
```

Comment:
```suggestion
### `fetch('hyper://NAME/$/diff/OLD_VERSION...NEW_VERSION/folder/')`
```

Github uses three dots so lets do the same

Reply:
We decided to use `..`